js/test/calendar/testhandate.js

```diff
@@ -2368,7 +2368,7 @@ module.exports.testhandate = {
         test.ok(hd !== null);
 
         // 3/1 is a Sunday, so it is at the beginning of the first week in China
-        test.equal(hd.getWeekOfMonth("zh-Hans-CN"), 1);
+        test.equal(hd.getWeekOfMonth("zh-Hans-CN"), 0);
```

Comment:
Why did this change? This shouldn't change... It is not affected by the weekend start/end calculations, or is it?

Reply:
The firstday of week for CN has changed. from Sunday to Monday. so I thought it was related test cases. because it shows an error after I updated the data..